Library/ViewModels/PledgeShippingLocationViewModel.swift

```diff
@@ -254,7 +254,7 @@ private func getRewardIDsToQuery(for project: Project) -> Set<Int> {
   if rewardIDsToQuery.isEmpty {
     let restrictedRewards = project.rewards
       .filter {
-        ($0.isRestrictedShippingPreference || $0.isLocalShippingPreference || $0.hasNoShippingPreference)
```

Comment:
local shipping items by default have shipping disabled. removing this check since it isn't a possible case here.